src/main/kotlin/org/rust/cargo/runconfig/run/CargoRun.kt

```diff
@@ -0,0 +1,25 @@
+package org.rust.cargo.runconfig.run
+
+import com.intellij.execution.configurations.RunConfiguration
+import com.intellij.execution.configurations.RunConfigurationModule
+import com.intellij.openapi.options.SettingsEditor
+import com.intellij.openapi.project.Project
+import org.rust.cargo.runconfig.CargoRunConfigurationBase
+import org.rust.cargo.runconfig.ui.CargoRunConfigurationEditForm
+
+/**
+ * [RunConfiguration] implementation that runs the main binary of a local package
+ */
+class CargoRun(project: Project,
```

Comment:
I think it's better to suffix all different configurations with `RunConfiguration`, so you'll have `CargoTestRunConfiguration` and `CargoRunRunConfiguration`. This is verbose, but helps when you "GoTo Class".